main/zoe/api/zoe-helpers.md

```diff
@@ -147,6 +147,53 @@ canTradeWith(leftInvite, rightInvite)
 canTradeWith(leftInvite, cantTradeRightInvite)
 ```
 
+## zoeHelpers.canTradeWithMapKeywords(leftOfferHandle, rightOfferHandle, keywords)
+- `leftOfferHandle`
+- `rightOfferHandle`
+- `keywords` - correspondence map of keywords on the left and right handles. keywords is an
+  array containing two equal-length arrays.
+- Returns: `{Boolean}`
+
+Checks if the `give` and `want` of two invites would satisfy offer
+safety if the two allocations are swapped mapping the keywords as specified.
+
+```js
+import { makeZoeHelpers } from '@agoric/zoe/src/contractSupport/zoeHelpers';
+
+const { canTradeWithMapKeywords } = makeZoeHelpers(zoe);
+
+const leftInvite = harden({
```

Comment:
I think this part is confusing. It's called Invite, but it's not an invite, it's the proposal. And the helper itself takes offerHandles - so there's a couple of things going on that would throw the user, I think.

Reply:
thanks. I made the handles in the example opaque.

Reply:
Looks good! 👍